scripts/add-gpgme-and-gnupg21-to-venv

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+GNUPG_DEPS="\
+    https://gnupg.org/ftp/gcrypt/npth/npth-1.3.tar.bz2 \
+    https://gnupg.org/ftp/gcrypt/libksba/libksba-1.3.5.tar.bz2 \
```

Comment:
I suspect the ordering not be good, see my script run failing : https://outils.crapouillou.net/zerobin/?a392b7647324166a#FnObJoiGlH1fZes3vcFklqtL97UmkOaAXEPu94axM9I=